Sources/SwiftPlot/PlotStyleHelpers.swift

```diff
@@ -42,3 +42,18 @@ public struct PlotLegend {
     public var textSize: Float = 10
     public init() {}
 }
+
+public protocol Annotation {
+    var text: String { get set }
+    var color: Color { get set }
+    var size: Float { get set }
```

Comment:
Instead of `var size: Float { get set }` I'd like it to be of the type `Size` which you can find in `Pair.swift`. This will be useful when you need to draw a rect or an arrow. Having a float size doesn't make sense for these applications. You can still have a `var textSize: Float` in `TextAnnotation`. In `TextAnnotation` the size can be a [computed property](https://medium.com/@micosmin/swift-computed-properties-2a8fd9f88d6). You will be able to get the bounding box/Size for a text from any renderer. Check Line 184 in Renderer.swift.

Reply:
With @karwa's suggestion of stripping down the Annotation protocol, would this still be applicable? If the Annotation protocol was stripped just to the ```draw``` function, would it make more sense to have ```size``` as a ```Float``` when defining ```Text```?

Reply:
yes